Makefile

```diff
@@ -126,3 +126,43 @@ regenerate-test-data: install-3.8
 	find examples -name "*.yaml" -type f -delete
 	HERA_REGENERATE=1 make test examples
 	@poetry run python -m pytest -k test_for_missing_examples --runxfail
+
+.PHONY: install-minikube
```

Comment:
We should use k3d instead. It is much quicker to spin up.

Reply:
Just for posterity (and the curious) - minikube took 41s [here](https://github.com/argoproj-labs/hera/actions/runs/7144548456/job/19458392050?pr=887), and k3d took 49s [here](https://github.com/argoproj-labs/hera/actions/runs/7170762330/job/19524240213?pr=887)

Reply:
Benchmarks from https://www.blueshoe.io/blog/minikube-vs-k3d-vs-kind-vs-getdeck-beiboot/ suggest k3d is faster and more efficient.